packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -0,0 +1,225 @@
+import {Box} from '@twilio-paste/box';
+import {
+  ChatAttachment,
+  ChatAttachmentDescription,
+  ChatAttachmentLink,
+  ChatBookend,
+  ChatBookendItem,
+  ChatBubble,
+  ChatEvent,
+  ChatLog,
+  ChatMessage,
+  ChatMessageMeta,
+  ChatMessageMetaItem,
+  useChatLogger,
+  ChatLogger,
+} from '@twilio-paste/chat-log';
+import {ChatComposer} from '@twilio-paste/chat-composer';
+import {AutoScrollPlugin} from '@twilio-paste/lexical-library';
+import {SendIcon} from '@twilio-paste/icons/esm/SendIcon';
+import {HelpText} from '@twilio-paste/help-text';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {Button} from '@twilio-paste/button';
+import {ButtonGroup} from '@twilio-paste/button-group';
+import {Stack} from '@twilio-paste/stack';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import {ChatDialogExample} from '../../../component-examples/ChatComposerExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/chat-composer"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/chat-composer/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Chat Composer"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/chat-composer"
+  storybookUrl="/?path=/story/components-chat-composer--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  noInline
+  language="jsx"
+  scope={{
+    Box,
+    ChatLog,
+    ChatBookend,
+    ChatBookendItem,
+    ChatEvent,
+    ChatMessage,
+    ChatBubble,
+    ChatMessageMeta,
+    Avatar,
+    ChatAttachment,
+    ChatAttachmentDescription,
+    ChatAttachmentLink,
+    ChatMessageMetaItem,
+    ChatComposer,
+    AutoScrollPlugin,
+    Button,
+    SendIcon,
+  }}
+>
+  {ChatDialogExample}
+</LivePreview>
+
+## Guidelines
+
+### About Chat Composer
+
+A Chat Composer is an input made for users to type rich chat messages. Chat Composer can be used as one part of larger chat user interfaces to provide a seamless authoring experience.
+
+### Accessibility
+
+Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component.
```

Comment:
suggestion: this is vague. Are there any aria attributes we want to suggest? We use aria-label in all the examples, it would be useful to detail that here. Accessibility is ultimately up to developers but this part of the docs is a good place to steer them in the right direction.